pkg/importer/spanner/sysl.arrai

```diff
@@ -0,0 +1,138 @@
+# size returns the size of an attribute
+let size = \length
+    cond {
+        length = 'MAX': '',
+        //seq.has_prefix('0x', length): '(' ++ //seq.trim_prefix('0x', length) ++ ')',
+        length > 0: $`(${length})`,
+        _: '',
+    };
+
+# attributePatternToken returns the token based on attribute value
+let attributePatternToken = \entity \attr \token
+    cond {
+        (attr.options?:{}): token,
+        (entity.primary_key where //seq.contains(attr.name, .)): token,
+        let fks = (entity.foreign_keys => ((.foreign_keys where .attribute = attr.name) =>
+            .reference_attribute orderby .)) where . != {}; (fks): token,
```

Comment:
Nit: all values except `{}`, `()` and `0` are equivalent to `true` in this context, so `!= {}` is redundant.

```suggestion
            .reference_attribute orderby .)) where .; (fks): token,
```

Reply:
ok